### PR TITLE
feat: use and initialize RDT Frontend in RDT panel

### PR DESCRIPTION
**Pull request by hoxyq**
# Summary

Stacked on https://github.com/facebookexperimental/rn-chrome-devtools-frontend/pull/22.

Last PR in this series of PRs. Uses [`react-devtools-fusebox`](https://github.com/facebook/react/tree/main/packages/react-devtools-fusebox) as a dependency to initialize React DevTools Frontend.

Loader is rendered while React DevTools is initialized:
- While we create global bindng and perform some initial Runtime.evaluate calls, **this can be throttled by current performance problems with initial load of source maps**

Potential improvements, will probably add later:
- Adding `catch` block in case if initialization failed, for displaying error message
- Removing worker CSP patch, and moving in to entrypoints build script, so only `rn_fusebox` is updated

# Test plan

Demo: 

https://github.com/facebookexperimental/rn-chrome-devtools-frontend/assets/28902667/cb0b1531-0d92-409c-a31f-5ac1f5526677

# Upstreaming plan

- [ ] This commit should be sent as a patch to the upstream `devtools-frontend` repo. I've reviewed the [contribution guide](https://docs.google.com/document/d/1WNF-KqRSzPLUUfZqQG5AFeU_Ll8TfWYcJasa_XGf7ro/edit#heading=h.9kj7femz1xg5).
- [x] This commit is React Native-specific and cannot be upstreamed.

### CODE REVIEW

File: front_end/panels/react_devtools/ReactDevToolsView.ts [hunk 1]
```diff
@@ -3,30 +3,100 @@
...
+    // Remove loader
+    this.contentElement.removeChildren();
+
+    const usingDarkTheme = window.matchMedia('(prefers-color-scheme: dark)').matches;
+
+    this.registerCSSFiles([ReactDevTools.CSS]);
+    ReactDevTools.initialize(this.contentElement, {
+      bridge: this.bridge,
+      store: this.store,
+      theme: usingDarkTheme ? 'dark' : 'light',
```
Comment (motiz88) [inline front_end/panels/react_devtools/ReactDevToolsView.ts:83]: Nits - can be filed as followup tasks IMO:
1. Is this reactive to theme changes?
2. Does CDT allow the user to override the theme? Will this honour that setting?

File: front_end/entrypoint_template.html [hunk 2]
```diff
@@ -14,7 +14,7 @@
     }
   }
 </style>
-<meta http-equiv="Content-Security-Policy" content="object-src 'none'; script-src 'self' 'unsafe-eval' https://chrome-devtools-frontend.appspot.com">
+<meta http-equiv="Content-Security-Policy" content="worker-src 'self' blob: ; object-src 'none'; script-src 'self' 'unsafe-eval' https://chrome-devtools-frontend.appspot.com">
```
Comment (motiz88) [inline front_end/entrypoint_template.html:17]: I wish this was more self-documenting. Might be worth an explanatory comment as part of the file.

**Review by motiz88 — APPROVED** [review]
Shipittttt

File: front_end/entrypoint_template.html (hunk 2 above)
Comment (hoxyq) [reply front_end/entrypoint_template.html:17]: I thought about moving this to `build_entrypoints` script, so only `rn_fusebox.html` entrypoint gets this.

Will add this tomorrow and add an explanatory comment, basically RDT frontend is using workers API for file uploads.

**Pull request merged**